backend/pkg/commons/types/frontend.go

```diff
@@ -89,9 +89,9 @@ const (
 	// Account dashboard events
 	IncomingTransactionEventName  EventName = "incoming_transaction"
 	OutgoingTransactionEventName  EventName = "outgoing_transaction"
-	ERC20TokenTransferEventName   EventName = "erc20_token_transfer"
-	ERC721TokenTransferEventName  EventName = "erc721_token_transfer"
-	ERC1155TokenTransferEventName EventName = "erc1155_token_transfer"
+	ERC20TokenTransferEventName   EventName = "erc20_token_transfer"   // #nosec G101
+	ERC721TokenTransferEventName  EventName = "erc721_token_transfer"  // #nosec G101
+	ERC1155TokenTransferEventName EventName = "erc1155_token_transfer" // #nosec G101
```

Comment:
The linter forbade me a lot of string names here because he assumed a password leak.
I set it to ignore it but this would also be a possibility I don't really like:
```	
tokenTransfer                 EventName = "token_transfer"
ERC20TokenTransferEventName   EventName = "erc20_" + tokenTransfer
ERC721TokenTransferEventName  EventName = "erc721_" + tokenTransfer
ERC1155TokenTransferEventName EventName = "erc1155_" + tokenTransfer

Reply:
The ignore is fine